src/clj/puppetlabs/services/ca/certificate_authority_service.clj

```diff
@@ -35,7 +42,19 @@
          auth-handler
          puppet-version)
        {:normalize-request-uri true})
-     (assoc context :auth-handler auth-handler)))
+     (watch-protocol/add-watch-dir! watcher
+                                    (fs/parent ca-crl-file)
+                                    {:recursive true})
```

Comment:
I left a similar comment in the related tk-jetty9 PR, https://github.com/puppetlabs/trapperkeeper-webserver-jetty9/pull/177#discussion_r127269511, about how it would probably be better here to look into enhancing the tk-filesystem-watcher service to be able to watch just the immediate parent directory for the CRL file rather than recursively watching the entire directory tree.